.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: Deploy release candidate
+on:
+    push:
+        branches:
+            - main
+    pull_request: # Runs on pull requests to any branch
+env:
+    IS_RELEASE_CANDIDATE: ${{ contains(github.head_ref, 'release--') }}
```

Comment:
Also we should say `BRANCH_PR` or `BRANCH_MERGE or COMMIT`

Reply:
done